ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SystemColumns.java

```diff
@@ -197,6 +197,14 @@ private static void validatePseudoColumnVersion(final int pseudoColumnVersionNum
     }
   }
 
+  /**
+   * This class was added with ROWPARTITION and ROWOFFSET to address the growing
+   * number of differences between the columns and ROWTIME. To add future PseudoColumns,
+   * one will need to consider each of the fields below, add a new pseudo column version
+   * (by incrementing CURRENT_PSEUDOCOLUMN_VERSION_NUMBER and setting the newest PseudoColumns
+   * to use the new CURRENT), and make the necessary changes in the logical schema and
+   * value transformer.
```

Comment:
```suggestion
   * to use the new CURRENT), and make the necessary changes in LogicalSchema.java and the
   * value transformers in SourceBuilder.java.
```

Reply:
I was hesitant about using class names given the possibility of these things changing, but I think that's unlikely and, empathizing with whoever has to do this in the future, having class names would be really nice. Commited